init.py

```diff
@@ -6,6 +6,6 @@
 while True:
     received_message = input("You: ")
     send_message, learn_response = message_to_bot(received_message, clf, learn_response)
-    print("MapBot:  " + send_message)
+    print("MapBot:", send_message)
```

Comment:
Letting Python deal with the `type` of `send_message` and always print it as a string.